test/slurm/job_conf.xml

```diff
@@ -26,7 +27,16 @@
           -->
           <param id="docker_volumes" from_environ="GALAXY_DOCKER_VOLUMES">$defaults</param>
         </destination>
+        <destination id="local" runner="local"/>
     </destinations>
+    <tools>
+        <!-- Tools can be configured to use specific destinations or handlers,
+             identified by either the "id" or "tags" attribute.  If assigned to
+             a tag, a handler or destination that matches that tag will be
+             chosen at random.
+         -->
+        <tool id="upload1" destination="local"/>
```

Comment:
wow, that's a long list ... I will have to check a few of them.
It's actually not that hard to run all of this on the cluster, with this PR we're also exporting the local tools, as well as galaxy's requirements.txt, so a `pip install -r /galaxy-central/requirements.txt --index-url https://wheels.galaxyproject.org/simple` from one of the cluster nodes should suffice.